### PR TITLE
Add docker things

### PR DESCRIPTION
**Pull request by hex337**
This adds the mostly standard docker set up.

`make build`
`make up`
`make up` <--- need to add a delay so that postgres has time to boot up

`make logs`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	touch .env
+	docker-compose build
+
+deps: #: Install the dependencies
+	docker-compose run --rm -e MIX_ENV=test $(SERVICE) mix deps.get
+
+format: #: Run mix format
+	docker-compose run --rm $(SERVICE) mix format
+
+iex: #: Interactive elixir shell on container
```
Comment (finack) [inline Makefile]: Eh?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+migrate: #: Run migrations
+	docker-compose run --rm $(SERVICE) mix ecto.migrate
+
+ps: #: Show running processes
+	docker-compose ps
+
+restart: #: Restart the service container
+	docker-compose restart $(SERVICE)
+
+seed: #: Seed the DB
```
Comment (finack) [inline Makefile]: Should we move this to `go run cmd/migration/main.go`?

**Review by finack — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (hex337) [reply Makefile]: Leftovers, whoops.

File: Makefile (hunk 2 above)
Comment (hex337) [reply Makefile]: We can definitely have a seed command. I just removed it for now though, and then we can figure out what we want later.

**Pull request merged**